pkgs/top-level/python-packages.nix

```
@@ -4629,6 +4629,8 @@ in {
 
   omegaconf = callPackage ../development/python-modules/omegaconf { };
 
+  opuslib = disabledIf (!isPy3k) (callPackage ../development/python-modules/opuslib { });
```

move this into the expresssion
```suggestion
  opuslib = callPackage ../development/python-modules/opuslib { };
```

---

thanks for the advice